.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
```

Comment:
Can we add a 

**Reference** (or other name)

field where people can add in papers/books/websites that can provide information on how to implement the feature in the way that the user wants? Or person to contact for more info?

Reply:
@wendycwong I can mention that in the "Additional context" section - how does that sound?

Reply:
Can you add

**Support ticket**  

as well.  We cannot disclose the customer but this link will bring us there and so we can raise the priority

Reply:
Sure you can add a field and tell users to add papers/books/web links

Reply:
I mean I'll just add that as a note to "Additional context" rather than making the form longer with more sections.